ampd/src/handlers/config.rs

```diff
@@ -47,6 +47,14 @@ pub enum Config {
         rpc_url: Url,
         rpc_timeout: Option<Duration>,
     },
+    MvxMsgVerifier {
+        cosmwasm_contract: TMAddress,
+        proxy_url: Url,
```

Comment:
why call this `proxy_url` versus `rpc_url`?

Reply:
Because it is not using RPC calls like EVM chains, but REST calls.

And `proxy` is the internal naming used by MultiversX for this service:
https://docs.multiversx.com/sdk-and-tools/proxy